roles/openshift_health_checker/meta/main.yml

```diff
@@ -2,3 +2,4 @@
 dependencies:
   - role: openshift_facts
   - role: openshift_repos
+  - role: openshift_version
```

Comment:
I had a look through `openshift_version` and I can live with it. If it fails, there's a pretty fundamental problem and the user should probably fix that first. The messaging doesn't look too bad coming through our callback plugin, but I drew up a [PR](https://github.com/openshift/openshift-ansible/pull/3949) to improve it a bit.

We do need to improve our messaging when docker and/or docker-py aren't available but that's unrelated to this line of code :)